caching-service/src/main/java/org/zowe/apiml/caching/model/KeyValue.java

```diff
@@ -51,6 +51,6 @@ public KeyValue() {
         key = "";
         value = "";
         serviceId = "";
-        created = "";
+        created = currentTime();
```

Comment:
Maybe i'm missing something here, please excuse me. How does the deserialization of values from storage works? Will this allow us to deserialize the stored date?

Reply:
I think that it will as the JSON should overwrite the data with the information obtained in the actual set. This is just default.

Reply:
And I have tested with the current VSAM implementation and the integration tests are passing with this setup.